services/movie_session.py

```diff
@@ -20,15 +19,17 @@
     return queryset
 
 
-def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
-    return MovieSession.objects.get(id=movie_session_id)
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = Ticket.objects.filter(
+        movie_session_id=movie_session_id).values("row", "seat")
+    return list(tickets)
 
 
 def update_movie_session(
     session_id: int,
     show_time: str = None,
     movie_id: int = None,
-    cinema_hall_id: int = None,
+    cinema_hall_id: int = None
 ) -> None:
     movie_session = MovieSession.objects.get(id=session_id)
```

Comment:
Using `get()` can raise a `DoesNotExist` exception if no `MovieSession` with the given `id` is found. Consider handling this exception to prevent runtime errors.